src/Tests/Application/app/AppKernel.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Sylius\Bundle\CoreBundle\Application\Kernel;
+use Symfony\Component\Config\Loader\LoaderInterface;
+
+final class AppKernel extends Kernel
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function registerBundles()
+    {
+        return array_merge([
+            new \Lakion\SyliusElasticSearchBundle\LakionSyliusElasticSearchBundle(),
```

Comment:
It probably can be placed together with other bundles, no need to include it as the first bundle.